network/network/src/main/java/bisq/network/p2p/node/network_load/NetworkLoadSnapshot.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.network.p2p.node.network_load;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.annotation.Nullable;
+
+@Slf4j
+@Getter
+@ToString
+@EqualsAndHashCode
```

Comment:
You can replace these by [@Value](https://projectlombok.org/features/Value) which would also allow you to remove the private's and the final's

Reply:
I find that omitting final and private reduces clarity. Specially for objects sent over the network having the class defined final is important for security. The @Value annotation would also trigger in IntelliJ a warning that final/private is not required.
I personally also prefer to have it more explicitly visible which annotation are added. In Bisq 1 more of the complex Lombok annotations are used but I find that sub-optimal and error prone(e.g. auto-constructors).